components/brave_ads/core/internal/serving/inline_content_ad_serving.cc

```diff
@@ -94,7 +94,6 @@ void InlineContentAdServing::BuildUserModelCallback(
     const std::string& dimensions,
     MaybeServeInlineContentAdCallback callback,
     const UserModelInfo& user_model) const {
-  CHECK(eligible_ads_);
```

Comment:
`CHECK` is not required because we call `IsSupported` within the flow of `MaybeServeAd`